Drop after FreeBSD 11 EOL, see https://svnweb.freebsd.org/changeset/base/303524

media/libcubeb/src/cubeb_oss.c:362:10: error: implicit declaration of function 'getline' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
  while (getline(&line, &linecap, sndstatfp) > 0) {
         ^
media/libcubeb/src/cubeb_oss.c:850:13: warning: comparison of integers of different signs: 'long' and 'unsigned int' [-Wsign-compare]
    if (nfr > s->bufframes) {
        ~~~ ^ ~~~~~~~~~~~~

--- media/libcubeb/src/cubeb_oss.c.orig	2020-12-07 23:35:05 UTC
+++ media/libcubeb/src/cubeb_oss.c
@@ -10,6 +10,9 @@
  * accompanying file LICENSE for details.
  */
 
+#if defined(__FreeBSD__) && __FreeBSD__ < 12
+#define _WITH_GETLINE
+#endif
 #include <assert.h>
 #include <ctype.h>
 #include <limits.h>
